nomos-services/network/src/backends/libp2p.rs

```diff
@@ -112,14 +149,8 @@ impl NetworkBackend for Libp2p {
                                 log_error!(swarm.connect(peer_id, peer_addr));
                             }
                             Command::Broadcast { topic, message } => {
-                                match swarm.broadcast(&topic, message) {
-                                    Ok(id) => {
-                                        tracing::debug!("broadcasted message with id: {id} tp topic: {topic}");
-                                    }
-                                    Err(e) => {
-                                        tracing::error!("failed to broadcast message to topic: {topic} {e:?}");
-                                    }
-                                }
+                                let msg = MixnetMessage { topic, message };
+                                log_error!(mixnet_relay.send(MixnetMsg::Send(msg.as_bytes())).await);
```

Comment:
What do you guys think about this strategy, using mixnet only for the first hop of gossipsub? This is one of strategies that we discussed. Although we haven't verify this startegy in details, I'm getting an impression that this will probably meet our privacy requirements, as Lighthouse guys https://github.com/logos-co/nomos-node/issues/273#issuecomment-1657545446.

Reply:
Yes, and for now is probably the easiest approach. So I think is the path to go.